src/Monster.cpp

```diff
@@ -1,28 +1,61 @@
 #include "Monster.h"
+#include <random>
 #include <iostream>
 
+std::random_device rd;
+std::mt19937 gen(rd());
+std::uniform_real_distribution<> dis(0.0, 1.0);
```

Comment:
This should probably at least have it's own namespace, if not a whole separate file.